Lib/asyncio/base_events.py

```diff
@@ -543,6 +549,26 @@ def _asyncgen_firstiter_hook(self, agen):
                     'asyncgen': agen
                 })
 
+    async def shutdown_default_executor(self):
+        """Schedule the shutdown of the default executor."""
+        self._executor_shutdown_called = True
+        if self._default_executor is None:
```

Comment:
This check is needed, otherwise numerous exceptions occur from ``test_asyncio``. There's nothing to shutdown if the default executor isn't set.